### PR TITLE
feature: User crud

### PR DESCRIPTION
**Pull request by Lucas-Malatchoumy**
## Features

- Register user
- Login user
- Get users
- Get user
- Update user
- Delete user

### CODE REVIEW

File: user/models.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
+    firstname = models.CharField(max_length=100)
+    lastname = models.CharField(max_length=100)
+    email = models.EmailField(max_length=254)
```
Comment (Maengdok) [inline user/models.py]: Une raison particulière de le mettre à 254 plutôt que 255?

File: user/normalizers.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+def users_normalizer(data):
+    result = []
+    for user in data:
+
+        item = {
+            'id': user['id'],
+            'firstname': user['firstname'],
+            'lastname': user['lastname'],
+            'email': user['email'],
+            'password': user['password'],
```
Comment (Maengdok) [inline user/normalizers.py]: Penser à retirer le renvoie de mot de passe dans la réponse du JSON

File: user/normalizers.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    return result
+
+
+def user_normalizer(data):
+    return {
+        'id': data.id,
+        'firstname': data.firstname,
+        'lastname': data.lastname,
+        'email': data.email,
+        'password': data.password,
```
Comment (Maengdok) [inline user/normalizers.py]: Pareil içi

File: user/urls.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+from . import views
+
+urlpatterns = [
+    path('add', views.register, name='register'),
+    path('', views.get_users, name='get_users'),
+    path('<int:user_id>', views.get_user, name='get_user'),
+    path('login', views.login, name='login'),
+    path('delete/<int:user_id>', views.delete_user , name='delete_user'),
+    path('update/<int:user_id>', views.update_user , name='update_user')
```
Comment (Maengdok) [inline user/urls.py:11]: Voir pour normaliser l'url de l'update

**Review by sylvaindnd — APPROVED** [review]

File: user/normalizers.py (hunk 3 above)
Comment (Lucas-Malatchoumy) [reply user/normalizers.py]: [c61108b](https://github.com/Axel-Pion-MDS/M1_Project_Timer_Back/pull/1/commits/c61108ba1afe7e48c198b23ef75f257287acfdf8)

File: user/normalizers.py (hunk 2 above)
Comment (Lucas-Malatchoumy) [reply user/normalizers.py]: [c61108b](https://github.com/Axel-Pion-MDS/M1_Project_Timer_Back/pull/1/commits/c61108ba1afe7e48c198b23ef75f257287acfdf8)

File: user/models.py (hunk 1 above)
Comment (Lucas-Malatchoumy) [reply user/models.py]: [c61108b](https://github.com/Axel-Pion-MDS/M1_Project_Timer_Back/pull/1/commits/c61108ba1afe7e48c198b23ef75f257287acfdf8)

**Review by Maengdok — APPROVED** [review]

**Pull request merged**